developer/src/kmc-kmn/package.json

```diff
@@ -61,6 +62,7 @@
     "exclude-after-remap": true,
     "exclude": [
       "src/import/",
+      "src/kmw-compiler",
```

Comment:
This will come back out at some point -- once we have coverage for kmw-compiler (if we ever get there).